src/nidigital/metadata/functions.py

```diff
@@ -989,6 +1003,8 @@
             },
             {
                 'direction': 'in',
+                'is_repeated_capability': True,
+                'repeated_capability_type': 'sites',
```

Comment:
In #1306, `pinList` was converted into a repeated capability. Since we can't have two unrelated repeated capabilities in the same method, we need to choose one over the other. I think we should choose `pinList` and revert `site` to being a parameter. Two reasons for this:
- API expects `site` in this method to represent exactly one site. Usability gain from making `sites` a repeated capability is pretty low.
- #1301 updates `get_pattern_pin_names` to return a `list` of `str`. Users typically would want use the output from that method for the `pin_list` in this method, which they can do easily if `pins` is a repeated capability. Else, they'll have to convert `list` of `str` into single `str`.

If we decide to keep `site` as a parameter, we should do the same for other HRAM methods, for the sake of consistency -  `FetchHistoryRAMCycleInformation` and `FetchHistoryRAMScanCycleNumber`.

Also, it would be nice if users can pass `site` as an int rather than as `str` formatted as `siteN`. I'm guessing we can do this with a simple converter. This should be done in a separate PR (please open an issue, if we decide to keep `site` as a parameter).

Reply:
What about `GetHistoryRAMSampleCount()` and `IsSiteEnabled()`? I looked for any method with a parameter name of 'site' instead of 'siteList'.

For now, I have added the converter and set the metadata to use it for all functions that take a single site (based on parameter name)

Reply:
`GetHistoryRAMSampleCount()` yes but `IsSiteEnabled()` no. This way the oddity (`site` is not a repeated capability) is restricted to HRAM API.